restcomm/restcomm.dao/src/main/java/org/restcomm/connect/dao/DaoUtils.java

```diff
@@ -199,4 +217,12 @@ public static String writeApplicationKind(Application.Kind kind) {
         }
     }
 
+    public static String writeGeolocationType(Geolocation.GeolocationType geolocationType) {
```

Comment:
You were wright and no plans to add it in the near future. After double checking and several manual tests, I removed it in commit a4d6463.